wallet/how-to/sign-data/connect-and-sign/unity.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 2
+---
+
+# Connect and sign in Unity
+
+You can [connect and sign](index.md) in a single interaction from your Unity game.
+
+## Prerequisites
+
```

Comment:
Maybe some of these are requirements? I may be splitting hairs.

Reply:
I'm mostly using vocabulary consistent with the rest of the MetaMask docs. We should discuss updating site-wide terms as a separate issue / a style guide update.